docs/demos/dns-policy/dnspolicy-demo.md

```diff
@@ -14,7 +14,13 @@
 
 ```bash
 ./install.sh
-(export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build-controller install run-controller)
+(export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build-gateway-controller install run-gateway-controller)
+
+## separate window
+
+(export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build-policy-controller install run-policy-controller)
```

Comment:
You can remove `(export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs)` from these commands, it's not been needed for a while since the controller no longer uses the env vars